diff/print.go

```diff
@@ -36,6 +37,16 @@ func PrintFileDiff(d *FileDiff) ([]byte, error) {
 		}
 	}
 
+	// FileDiff is added/deleted file
+	// No further hunks printing needed
+	if d.NewName == "" {
```

Comment:
If you do the changes as suggested in parse.go, then you probably need to update this here too to also catch `if d.OldName == ""`.

Reply:
All filenames from `Only in {source}: {filename}` messages will be added to `OrigName` field.